deps/prebuilt/include/TAEF/WexTestClass.h

```diff
@@ -587,17 +622,25 @@ namespace WEX
 #pragma section("testdata$g_TCM", read)
 #pragma section("testdata$h_TMM", read)
 
+#define TAEF_ATTRIBUTE_SECTION(sectionName)                 __declspec(allocate(sectionName))
+
 // The IntelliSense compiler reports errors when trying to parse 'TAEF_PIN_FUNCTION_SYMBOL', so it should be skipped.
 #if defined(__INTELLISENSE__)
-# define TAEF_PIN_FUNCTION_SYMBOL
+#define TAEF_PIN_FUNCTION_SYMBOL
```

Comment:
So I thought that my code formatter was going wacky here. I'm fine putting it back but I wasn't aware of any "scoping" convention here. # define looks like a syntax error at first glance. Would it be more clear to do 

#if (...)
    #define ...

as a scoping mechanism? 

I'm also fine punting this just to minimize churn though.

---
In reply to: [92519227](https://github.com/Microsoft/WinObjC/pull/1523#discussion_r92519227) [](ancestors = 92519227)